rcon/recorded_commands.py

```diff
@@ -53,7 +54,25 @@ def thread_pool(self):
     def run_in_pool(self, function_name: str, *args, **kwargs):
         return self.thread_pool.submit(getattr(self, function_name), *args, **kwargs)
 
-    @mod_users_allowed
+    @ttl_cache(ttl=2)
```

Comment:
not sure if it's wise to cache that, the calls inside it are already cached no? and now you should invalidate on kick/ban/move player